packages/protocol/test/L1/TaikoL1TestBase.sol

```diff
@@ -24,6 +24,17 @@ abstract contract TaikoL1TestBase is TaikoTest {
     GuardianProver public gp;
     TaikoA6TierProvider public cp;
     Bridge public bridge;
+    
+    // For SGX remote attestation
+    AutomataDcapV3Attestation attestation;
```

Comment:
we should not add subclass variables to the base, the base will get larger and larger...